rclcpp/include/rclcpp/experimental/buffers/ring_buffer_implementation.hpp

```diff
@@ -91,6 +95,7 @@ class RingBufferImplementation : public BufferImplementationBase<BufferT>
     }
 
     auto request = std::move(ring_buffer_[read_index_]);
+    TRACEPOINT(ring_buffer_dequeue, static_cast<const void *>(this), read_index_, size_-1);
```

Comment:
```suggestion
    TRACEPOINT(ring_buffer_dequeue, static_cast<const void *>(this), read_index_, size_ - 1);
```

Reply:
fixed in [9ead575](https://github.com/ros2/rclcpp/pull/2091/commits/9ead5753751ed694c7acac47483c6f37ef61919d).